README.md

```diff
@@ -0,0 +1,25 @@
+# The Website of software-developer.org
+
+A site containing blogs about technologies and software development, adressed to developers.
+
+# Getting Started
+
+## Installation
+
+Run `npm install` to install all packages.
+
+## Run Website
+
+Run website you can:
+
+```bash
+# either start ...
+npm start
+# ... or start in debug mode
+DEBUG=website:* npm start
+# now open localhost:3000 in your browser
+```
```

Comment:
A chapter for folder structure would be good. Also maybe some resources to express. Also maybe a simple explanation about the key files (e.g. templates).